### DISCUSSION TITLE
Can I install and run this on Linux?

### DISCUSSION THREAD

**Discussion by rahul-dev-app** [AdriaGual/marvel-snap-bot#56] [Q&A] [answered]
Will I be able to use a Linux alternative for BS like Anbox or Genymotion and run the bot on a Linux machine?

**Comment by AdriaGual** [comment, marked as answer]
Hmm actually I haven't tried this BUT, if you can do the 'adb devices' command and it returns you the emulator and the port, you just have to set the adb_path to that folder and change the port (5555 is the one for BS but Anbox or Genymotion will use another one for sure).

Hope it helps!

**Reply by rahul-dev-app** [reply]
Thank you so much for the response!

I am able to get the adb_path, but I think genymotion (free version at least) does not allow me to emulate an device version advanced enough for me to get Snap going. I'm gonna give it a try again, but for now I'm resorting to BS-X.
